guide/client_service/client_main.bal

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/http;
+import ballerina/log;
+
+endpoint http:Client studentData {
+    url: " http://localhost:9292"
+};
+
+function main(string... args) {
+    http:Request req = new;
+    int operation = 0;
+    while (operation != 6) {
+        // print options menu to choose from
+        io:println("Select operation.");
+        io:println("1. Add student");
+        io:println("2. View all students");
+        io:println("3. Delete a student");
+        io:println("4. Make a mock error");
+        io:println("5: Get a student's marks");
+        io:println("6. Exit");
+        io:println();
+
+        string choice = io:readln("Enter choice 1 - 5: ");   // read user's choice
```

Comment:
Better to add the comment on the line above. Please fix in other places as well.